src/_tests_/Navabr.test.js

```diff
@@ -0,0 +1,19 @@
+import { MemoryRouter } from 'react-router-dom';
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import Navbar from '../components/navbar/Navbar';
+
+afterEach(() => {
+  cleanup();
+});
+
+describe('Navbar', () => {
+  it('renders navbar', () => {
+    const tree = render(
+      <MemoryRouter>
+        <Navbar />
+      </MemoryRouter>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Great Job using React Testing Library to test your components, but as part of the requirements you should add some unit tests using jest as well, kindly add them 🙏